Apps/APN-UIKit/APN UIKit/Storage/UserDefaultKeys.swift

```diff
@@ -1,9 +1,10 @@
 import Foundation
 
 enum UserDefaultKeys: String {
-    case trackUrl
+    case cdnHost
```

Comment:
Do we need to add cdn host to the sample app? Are we planning on ever changing it during QA testing?

Reply:
TrackUrl has been _kind of_ broken down into two - cdnHost, apiKey. It is not clear if we plan to change these values during QA testing but I added these two fields as a replacement to trackUrl.